app/Http/Controllers/BeatmapsetsController.php

```diff
@@ -402,8 +403,10 @@ private function showJson($beatmapset)
             'beatmaps',
             'beatmaps.failtimes',
             'beatmaps.max_combo',
+            'beatmaps.owners',
             'converts',
             'converts.failtimes',
+            'converts.owners',
```

Comment:
as a bonus(?), if just need the user ids, it can potentially be selected together with the beatmap itself with something like in https://github.com/ppy/osu-web/blob/1fbc73baa8e8be6759b1cc4f4ab509d8ae53a165/app/Models/Multiplayer/Room.php#L296-L321

Reply:
`owners` is included on `converts` so the UI side doesn't have to go digging the for the non-convert to check for mappers.

Reply:
`owners` is also contains the minimum to display on pages that don't have `related_users` like the score page instead of a separate include for just the ids, otherwise, the score page should sideload the users instead.

Reply:
> `owners` is included on `converts` so the UI side doesn't have to go digging the for the non-convert to check for mappers.

just include the ids then?

> `owners` is also contains the minimum to display on pages that don't have `related_users` like the score page instead of a separate include for just the ids, otherwise, the score page should sideload the users instead.

include it only on that specific page then? or add the related users.

seems like a lot of data being returned for everything else just to deal with single page with its single beatmap

Reply:
`owners` is just the id and username...

Reply:
I'm going to remove the duplicated data from `converts` but it may or not be part of this PR since it needs some extra type and client-side beatmap lookup changes